RequestManagementSystem/Client/Request.py

```diff
@@ -85,6 +86,8 @@ def __init__( self, fromDict = None ):
     self.RequestName = None
     self.OwnerGroup = None
     self.SourceComponent = None
+    
+    self.dmsHelper = DMSHelpers())
```

Comment:
Ugly coupling, but we have no choice :-( 
But I repeat again: soon (very soon), this optimization based on failover will have to go to the failoverModule, and not here. And then the coupling will be gone :-)